ansys/mapdl/core/mapdl.py

```diff
@@ -1364,7 +1364,7 @@ def kplot(
         return super().kplot(np1=np1, np2=np2, ninc=ninc, lab=lab, **kwargs)
 
     @property
-    def result(self):
+    def result(self) -> 'Result':
```

Comment:
```suggestion
    def result(self) -> Result:
```

Reply:
I believe an class object should be specified if we are talking about Hinting in Python. I guess PyCharm will be consistent with this.

Reply:
The quotes are meant for forward references, which I originally thought was needed for this function, but with the import I guess it is not.